bin/install.ps1

```diff
@@ -330,40 +430,56 @@ Function Install-Requirements {
     if ($vc_exists) {
       $(Get-ProcEnvs($vc_vars))
     } else {
-      Write-Output "vcvars64.bat still not present, something went wrong."
-      Exit 1
+      if ($env:CI -eq $null) {
+        $global:install_errors += "vcvars64.bat still not present, something went wrong."
```

Comment:
should this bail here?

Reply:
This function gets called at the end of the script. It allows us to report multiple issues instead of failing on a single issue, as well as only return a bad status code depending on the context of how the script was called.

https://github.com/socketsupply/socket/pull/219/files#diff-e9a96fb537f64b60e955f6d260d0d3c697cb45050752cf4d834833486b4d8530R46-R53